pkg/rehearse/jobs.go

```diff
@@ -517,17 +517,20 @@ type periodicsByName map[string]prowconfig.Periodic
 type presubmitsByName map[string]prowconfig.Presubmit
 
 // selectJobsForRegistryStep returns a sample from all jobs affected by the provided registry node.
-func selectJobsForRegistryStep(node registry.Node, configs []*config.DataWithInfo, allPresubmits presubmitsByName, allPeriodics periodicsByName, skipJobs sets.String) (presubmitsByRepo, periodicsByRepo) {
+func selectJobsForRegistryStep(node registry.Node, configs []*config.DataWithInfo, allPresubmits presubmitsByName, allPeriodics periodicsByName, skipJobs sets.String, loggers Loggers) (presubmitsByRepo, periodicsByRepo) {
 	selectedPresubmits := make(map[string][]prowconfig.Presubmit)
 	selectedPeriodics := make(map[string][]prowconfig.Periodic)
 
+	loggers.Debug.WithField("node-name", node.Name()).Debug("Searching for jobs affected by changed node")
 	for _, cfg := range configs {
+		logFields := cfg.Info.LogFields()
```

Comment:
We don't generally do this, but instead `WithFields()` right away to add to the logger. Why keep the fields around without adding them in this case?

Reply:
Will fix